wlib/PreferenceDlg.h

```diff
@@ -23,6 +23,7 @@ static const TCHAR EditorSection[](_T("Editor"));
 static const SectionLabelDefault GLOBAL_MAINTAININDENT(SectionLabel(EditorSection, _T("MaintainIndent")), 1);
 static const SectionLabelDefault GLOBAL_TAB_WIDTH(SectionLabel(EditorSection, _T("TabWidth")), 2);
 static const SectionLabelDefault GLOBAL_TAB_USESPACES(SectionLabel(EditorSection, _T("UseSpaces")), true);
+static const SectionLabelDefault GLOBAL_BOM(SectionLabel(EditorSection, _T("BOM")), true);
```

Comment:
This will need to be declared in comms and #include here,

Reply:
Now in global.h